src/UWP/ArcGISRuntime.UWP.Viewer/Samples/GeometryEngine/Buffer/Buffer.xaml.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using System;
+using Windows.UI.Popups;
+
+namespace ArcGISRuntime.UWP.Samples.Buffer
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Buffer",
+        "GeometryEngine",
+        "This sample demonstrates how to use the GeometryEngine.Buffer to generate a polygon from an input geometry with a buffer distance.",
+        "Tap on the map to specify a map point location. A buffer will created and displayed based upon the buffer value (in miles) specified in the textbox. Repeat the procedure to add additional map point and buffers. The generated buffers can overlap and are independent of each other.",
+        "")]
+    public partial class Buffer
+    {
+        // Graphics overlay to display buffer related graphics.
```

Comment:
I think this needs a hyphen between 'buffer' and 'related'. Applies to other platforms as well.